### PR TITLE
Goovirus

### PR DESCRIPTION
**Pull request by reforger**
Adds two classes called SecretSauce and FortuneTell, creating listeners for both.

### CODE REVIEW

File: src/main/java/org/gooteam/FortuneTell.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        String[] FailureString = {
+                "Fortune telling failed! Fuck!",
+                "Epic Fail!",
+                "EPIC FAIL!!!",
+        };
+
+        double FortuneFailChance = Math.random();
+
+        if(FortuneFailChance < 0.3) {
+            int FailureRoll = (int) Math.round(Math.random() * 2);
```
Comment (hingler) [inline src/main/java/org/gooteam/FortuneTell.java]: biased towards the entry at index 1 (0.5 - 1.5) - floor(Math.random() * 3) will be equal dist (applies to other cases as well)

File: src/main/java/org/gooteam/FortuneTell.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                "PERFECT ALIGNMENT",
+        };
+
+        String[] FailureString = {
+                "Fortune telling failed! Fuck!",
+                "Epic Fail!",
+                "EPIC FAIL!!!",
+        };
+
+        double FortuneFailChance = Math.random();
```
Comment (hingler) [inline src/main/java/org/gooteam/FortuneTell.java:43]: first letter of var names should be lowercase - doesn't matter for this though

File: src/main/java/org/gooteam/FortuneTell.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+        if(FortuneFailChance < 0.3) {
+            int FailureRoll = (int) Math.round(Math.random() * 2);
+            ResultMessage = FailureString[FailureRoll];
+        }
+        else{
+            int FortuneRoll = (int) Math.round(Math.random() * 6);
```
Comment (hingler) [inline src/main/java/org/gooteam/FortuneTell.java]: gaussian here or no?

**Review by hingler — CHANGES_REQUESTED** [review]
couple comments look good otherwise

File: src/main/java/org/gooteam/FortuneTell.java (hunk 3 above)
Comment (reforger) [reply src/main/java/org/gooteam/FortuneTell.java]: let it be done.

File: src/main/java/org/gooteam/FortuneTell.java (hunk 2 above)
Comment (reforger) [reply src/main/java/org/gooteam/FortuneTell.java:43]: will keep it in mind

File: src/main/java/org/gooteam/FortuneTell.java (hunk 1 above)
Comment (reforger) [reply src/main/java/org/gooteam/FortuneTell.java]: don't get it yet bc I'm retarded but you can make the change now or I'll figure it out later

Comment (hingler) [reply src/main/java/org/gooteam/FortuneTell.java]: yeah sec

**Comment by hingler** [comment]
pushed the round -> floor change - lmk if/when you get gaussian sampling working

**Review by hingler — APPROVED** [review]

**Pull request merged**